TreeDB/cmd/treedb_rag_benchmark/service_overhead.go

```
@@ -0,0 +1,373 @@
+		P99Millis:  p99,
+		MeanMillis: mean,
+		Hits:       hits,
+	}
+}
+
+func postServiceOverheadJSON(client *http.Client, url string, body any, out any) error {
+	raw, err := json.Marshal(body)
+	if err != nil {
+		return err
+	}
+	resp, err := client.Post(url, "application/json", bytes.NewReader(raw))
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Post must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)